lib/discourse_plugin_registry.rb

```diff
@@ -209,6 +210,10 @@ def self.register_seedfu_filter(filter = nil)
     self.seedfu_filter << filter
   end
 
+  def self.register_permitted_bulk_action_parameters(filter = nil)
+    self.permitted_bulk_action_parameters << filter
+  end
+
```

Comment:
There shouldn't be any need to define this function. `define_register` will create it automatically